src/ssl.c

```diff
@@ -11328,15 +11328,16 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
 
     long wolfSSL_CTX_set_options(WOLFSSL_CTX* ctx, long opt)
     {
-        WOLFSSL *ssl;
+        WOLFSSL ssl;
```

Comment:
This puts a lot of memory on the stack.  Why the change from dynamic?

Reply:
@toddouska : The `wolfSSL_new()` function call was failing with `Server missing certificate` (https://github.com/wolfSSL/wolfssl/blob/master/src/internal.c#L4259) because the `ctx` did not yet have the server certificate loaded.

To resolve the increased stack I've pushed a fix to allocate the `ssl` from the heap.